src/lib.rs

```diff
@@ -96,7 +102,15 @@ pub fn run_program_with_custom_state(bin_path: &str, mut vm: VMState) -> (U256,
                     // hooked up.
                     break;
                 }
-                Variant::UMA(_) => todo!(),
+                Variant::UMA(uma_variant) => match uma_variant {
+                    UMAOpcode::HeapRead => _heap_read(&mut vm, &opcode),
+                    UMAOpcode::HeapWrite => _heap_write(&mut vm, &opcode),
+                    UMAOpcode::AuxHeapRead => _aux_heap_read(&mut vm, &opcode),
+                    UMAOpcode::AuxHeapWrite => _aux_heap_write(&mut vm, &opcode),
+                    UMAOpcode::FatPointerRead => _fat_pointer_read(&mut vm, &opcode),
+                    UMAOpcode::StaticMemoryRead => todo!(),
```

Comment:
In issue #12, include the `StaticMemoryRead` and `StaticMemoryWrite` opcodes. 
I assume you will do this in another PR, but just to confirm.

Reply:
In the VM specification there is no mention of StaticMemoryRead or StaticMemoryWrite, so they won't be implemented.
Also, on the vm2, it is marked as an unimplemented instruction.